Tests/ComposableArchitectureTests/FileStorageTests.swift

```diff
@@ -251,7 +251,7 @@ final class FileStorageTests: XCTestCase {
 
       try fileStorage.save(Data(), .fileURL)
       scheduler.run()
-      expectNoDifference(users, [])
+      expectNoDifference(users, [.blob])
```

Comment:
Changes made to the file on disk will be ignored if there is an enqueued state change waiting to be saved.